plugins/ltac2/g_ltac2.mlg

```diff
@@ -161,10 +161,10 @@ GRAMMAR EXTEND Gram
     | "4" LEFTA [ ]
     | "3" [ e0 = SELF; ","; el = LIST1 NEXT SEP "," ->
         { let el = e0 :: el in
-          CAst.make ~loc @@ CTacApp (CAst.make ~loc @@ CTacCst (AbsKn (Tuple (List.length el))), el) } ]
```

Comment:
Can't we put the loc of the strings `","` and `";"` there?

Reply:
My concern is not the doc, but error highlighting in case of type mismatch. I haven't tried your PR yet, but does it give a reasonable output in this area?

Reply:
Do you have something specific in mind? For example, the following behaves the same before and after the pull request.
```coq
Ltac2 foo () := ((0,1) : int*int*int).
```